pypugjs/testsuite/cases/text.html

```diff
@@ -1,4 +1,4 @@
-<option value="">-- (selected) --</option>
```

Comment:
we'll have to do similar patch for lots of test fixtures (as original pug quote is ' ). I want to know if this is fine first

Reply:
well, this doesnt sound right .. the quote char, whatever it is, shall be untouched and thus not breaking existing tests or are the tests putting ' in and expect " to come out?

I can probably could have a deeper look tonight.

Reply:
> are the tests putting ' in and expect " to come out?

Right. I think the test are incorrect due to the original behaviour was incorrect.
I would expect pypugjs should behave as much as possible the same as pugjs.

Even if probably this behaviour of keeping the quote char is not "specified".

I can try to run the test suite against pugjs to prove that point.